makefiles/deprecated_modules.inc.mk

```diff
@@ -2,6 +2,7 @@
 # Keep this list ALPHABETICALLY SORTED!!!!111elven
 DEPRECATED_MODULES += event_thread_lowest
 DEPRECATED_MODULES += gnrc_netdev_default
+DEPRECATED_MODULES += gnrc_netif_cmd_lora # use shell_cmd_gnrc_netif_lorawan instead
```

Comment:
c31e3732147c1f057c0d141ea4efc5d97f854f3c introduced a catch-all (`gnrc_netif_cmd_%`). I think that catch-all should be used here for deprecation as well.

Reply:
> [c31e373](https://github.com/RIOT-OS/RIOT/commit/c31e3732147c1f057c0d141ea4efc5d97f854f3c) introduced a catch-all (`gnrc_netif_cmd_%`). I think that catch-all should be used here for deprecation as well.

Ping?

Reply:
But after all this PR deprecates `gnrc_netif_cmd_lora` by renaming it. If there are no other modules matching `gnrc_netif_cmd_%`, maybe we should instead change `pseudomodules.mk` to no longer match with wildcards.

There is a check for typos in module names. But that check is not as effective as it could be due to all the wildcards. IMO we should get rid of all wildcards in `pseudomodules.mk`.

Reply:
> There is a check for typos in module names. But that check is not as effective as it could be due to all the wildcards. IMO we should get rid of all wildcards in `pseudomodules.mk`.

You won't get resistance on that from me :-). Then for this PR, should we just replace the `gnrc_netif_cmd_%` wildcard with `gnrc_netif_cmd_lora` to get this along? The actual removal of wildcards should be done in a separate PR IMHO.